config/application.rb

```diff
@@ -113,6 +104,8 @@ class Application < Rails::Application
       ::Time,
     ]
 
+    # Adds classes that can be used with YAML.safe_load for the versions table used by paper_trail
```

Comment:
This relates to the block above (yaml_column_permitted_classes) so needs to be moved back to where it was.